ResourceStatusSystem/Client/SiteStatus.py

```diff
@@ -179,7 +179,27 @@ def getUsableSites( self, siteNamesList ):
     """
 
     if not siteNamesList:
-      return S_ERROR(DErrno.ERESUNK, 'siteNamesList is empty')
+      if self.rssFlag:
+        activeSites = self.rsClient.selectStatusElement( 'Site', 'Status', status = 'Active', meta = { 'columns' : [ 'Name' ] } )
+        if not activeSites['OK']:
+          return activeSites
+
+        degradedSites = self.rsClient.selectStatusElement( 'Site', 'Status', status = 'Degraded', meta = { 'columns' : [ 'Name' ] } )
+        if not degradedSites['OK']:
+          return degradedSites
+
+        return activeSites['Value'] + degradedSites['Value']
+
+      else:
+        activeSites = self.wmsAdministrator.getSiteMask('Active')
```

Comment:
No, this is a mistake: `getSiteMask` don't take any parameter: https://github.com/DIRACGrid/DIRAC/blob/integration/WorkloadManagementSystem/Service/WMSAdministratorHandler.py#L68

So, here you can only do:

```
else:
  res = self.wmsAdministrator.getSiteMask()
  if not res['OK']:
    return res
```